lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -289,7 +294,9 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
         specimen_received = (collection_method == CollectionMethod.SWAB_AND_SEND and \
             is_complete('post_collection_data_entry_qc', redcap_record_instance)) or \
             (collection_method == CollectionMethod.KIOSK and \
-            is_complete('kiosk_registration_4c7f', redcap_record_instance))
+            is_complete('kiosk_registration_4c7f', redcap_record_instance)) or \
+            (collection_method == CollectionMethod.UW_DROPBOX and \
+            is_complete('huksy_test_kit_registration', redcap_record_instance))
```

Comment:
Looks like this should be `husky_test_kit_registration`.

Reply:
Thanks! Typo fixed.